docs/content/contribute/release-process.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Release process
+---
+
+# Release process
+
+The https://github.com/openshift/hypershift repo produces two different artefacts: Hypershift Operator (HO) and Control Plane Operator (CPO).
+
+The CPO release lifecycle is dictated by the [OCP release payload](https://access.redhat.com/support/policy/updates/openshift).
+
+The HO has an independent release cadence. For consumer products:
+- Our internal image build system builds from our latest commit in main several times a day.
+- To roll out a new build we apply the following process:
```

Comment:
Was this supposed to be in a list format? It's all smushed together in the preview - https://deploy-preview-2236--hypershift-docs.netlify.app/contribute/release-process/